docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
         timeout: 10s
         retries: 10
   cartography:
-    image: ghcr.io/lyft/cartography
+    image: lyft/cartography
```

Comment:
What's this do?

Reply:
Good catch, I changed the docker-compose file to run tests with local build.
I revert this change immediately.